certbot/compat/path.py

```diff
@@ -0,0 +1,31 @@
+"""This compat module wraps os.path to forbid some functions."""
+# pylint: disable=function-redefined
+from __future__ import absolute_import
+
+# First round of wrapping: we import statically all public attributes exposed by the os.path
```

Comment:
There's a cruel, nerdy, overachieving/overengineering part of me that wants to try and generalize this code of wrapping a module so it's not duplicated but it's not worth it.

Reply:
Well I would have really liked to do so, but on top of the complexity of abstracting this, you would lose a great feature: Python IDE would not be able to autocomplete anything, since the wrapper modules would become completely dynamic. Pylint and MyPy would become really angry also.